annotations/src/main/java/io/quarkiverse/operatorsdk/annotations/CSVMetadata.java

```diff
@@ -91,6 +91,12 @@
         boolean supported() default true;
     }
 
+    /**
+     * Additional RBAC rules that need to be provided because they cannot be inferred automatically. Note that RBAC rules added
+     * to your reconciler via {@link RBACRule} should already be handled automatically, under the service account name
+     * associated with your Reconciler so this annotation should only be used to add additional rules to other service accounts
+     * or for rules that you don't want to appear in the generated Kubernetes manifests.
+     */
```

Comment:
Kudos for this. Javadocs should be used way more :) 

@metacosm one thing I am still curious about is this being part of the `CSVMetadata`. Shouldn't this also have an impact when CSV bundle is not used? AFAIK the `CSVMetadata` class really is used just for CSV metadata generation, or am I wrong?

Reply:
`RBACRule` is not part of `CSVMetadata` and we still need permission rules on `CSVMetadata` in case you don't want something to be generated in the kubernetes manifests. Or am I misunderstanding your question?

Reply:
> `RBACRule` is not part of `CSVMetadata` and we still need permission rules on `CSVMetadata` in case you don't want something to be generated in the kubernetes manifests. Or am I misunderstanding your question?
I see.. That makes sense. So you would use this when you don't want the permission in your `kubernetes.yml` but still want to use it for CSV. Thanks!